scripts/install/requestrr.sh

```diff
@@ -0,0 +1,178 @@
+#!/bin/bash
+# Requestrr installation
+# Author: Brett
+# Copyright (C) 2021 Swizzin
+# Licensed under GNU General Public License v3.0 GPL-3 (in short)
+#
+#   You may copy, distribute and modify the software as long as you track
+#   changes/dates in source files. Any modifications to our software
+#   including (via compiler) GPL-licensed code must also be made available
+#   under the GPL along with build & install instructions.
+#shellcheck source="sources/functions/os"
+. /etc/swizzin/sources/functions/os
+
+echo_progress_start "Downloading source files"
+case "$(_os_arch)" in
+    "amd64") dlurl=$(curl -sNL https://api.github.com/repos/darkalfx/requestrr/releases/latest | grep -Po 'ht(.*)linux-x64(.*)zip') >> ${log} 2>&1 ;;
+    "armhf") dlurl=$(curl -sNL https://api.github.com/repos/darkalfx/requestrr/releases/latest | grep -Po 'ht(.*)linux-arm(.*)zip') >> ${log} 2>&1 ;;
+    "arm64") dlurl=$(curl -sNL https://api.github.com/repos/darkalfx/requestrr/releases/latest | grep -Po 'ht(.*)linux-arm64(.*)zip') >> ${log} 2>&1 ;;
+    *)
+        echo_error "Arch not supported"
+        exit 1
+        ;;
+esac
+
+if ! curl "$dlurl" -L -o /tmp/requestrr.zip >> "$log" 2>&1; then
+    echo_error "Download failed, exiting"
+    exit 1
+fi
+echo_progress_done "Source downloaded"
+
+echo_progress_start "Extracting archive"
+unzip -q /tmp/requestrr.zip -d /opt/ >> "$log" 2>&1
+rm /tmp/requestrr.zip
+mv /opt/requestrr* /opt/requestrr
```

Comment:
If you get the tar, you can just extract it with strip-component flag to get rid of the first dir. This could get hacky and wonky

Reply:
I didn’t see the dev has a tar available. Seems like zips only from the [Releases Page](https://github.com/darkalfx/requestrr/releases/tag/V1.0.17).

Originally I did want to go and use a tar for this reason. Unfortunately it doesn’t seem to be an option at this time.